tests/pax/nightly/common.libsonnet

```diff
@@ -0,0 +1,61 @@
+local common = import '../common.libsonnet';
+
+{
+  NightlyPaxTest:: common.PaxTest {
+    local config = self,
+    expPath:: '',
+    extraFlags:: '',
+    buildDate:: '$(date +%Y%m%d)',
+
+    // PAX tests are structured as bash scripts that run directly on the Cloud
+    // TPU VM instead of using docker images
+    testScript:: |||
+      set -x
+      set -u
+      set -e
+
+      # .bash_logout sometimes causes a spurious bad exit code, remove it.
+      rm .bash_logout
+
+
+      # check for nightly build
+      gsutil cp gs://pax-on-cloud-tpu-project/wheels/%(buildDate)s/paxml*.whl .
+      gsutil cp gs://pax-on-cloud-tpu-project/wheels/%(buildDate)s/praxis*.whl .
+
+      if [ -f praxis*.whl -a -f paxml*.whl ]; then
+          echo "Nightly builds succeeded."
+      else 
+          echo "Nighlty builds failed or are pending."
+          exit 1
+      fi 
+
+      # install pax and dependencies
+      pip install -U pip
```

Comment:
I don't recommend updating pip with pip. Do you need a more recent version of `pip` than is on the base image?

Reply:
I don't think so